pkgs/top-level/all-packages.nix

```diff
@@ -25683,8 +25683,6 @@ with pkgs;
     inherit (xorg) libSM;
   };
 
-  clickshare-csc1 = callPackage ../applications/video/clickshare-csc1 { };
```

Comment:
You probably want to add a `clickshare-csc1 = throw "...";` line to `pkgs/top-level/aliases.nix` as well.